applications/CompressiblePotentialFlowApplication/tests/naca0012_small_compressible_test/cl_jump.dat

```diff
@@ -0,0 +1 @@
+ 0.349746856119
```

Comment:
Same comment.

Reply:
This is solved in https://github.com/KratosMultiphysics/Kratos/pull/4668